climakitae/core/data_load.py

```diff
@@ -545,6 +546,20 @@ def _merge_all(selections, data_dict):
         output data
 
     """
+    # Two LOCA2 simulations have their hourly time coordinate shifted 12HR beyond the rest of the simulations
+    # Here we reindex the time dimension to shift it by 12HR for the two troublesome simulations
+    # This avoids the issue where every other day is set to NaN when you concat the datasets!
+    if (selections.downscaling_method in ["Statistical", "Dynamical+Statistical"]) and (
+        selections.timescale == "daily"
+    ):
+        troublesome_sims = ["HadGEM3-GC31-LL", "KACE-1-0-G"]
+        for sim in troublesome_sims:
+            for dset_name in list(data_dict):
+                if sim in dset_name:
+                    data_dict[dset_name]["time"] = data_dict[dset_name].time.get_index(
+                        "time"
+                    ) + timedelta(hours=12)
```

Comment:
I believe we want this to be `hours=-12` to shift 12 hours back, not forward. Need to confirm with the first timestamp, so that the data starts on e.g. Jan 1 midnight, rather than Jan 2 midnight

Reply:
Or, data_dict[dset_name]["time"] = data_dict[dset_name].time.get_index("time") - timedelta(hours=12)

Whichever is more appropriate in this case

Reply:
After conversation with @nicolejkeeney this comment can now be ignored; I had mistakenly thought we were shifting to midnight, not noon